docs/protocol.md

```diff
@@ -126,25 +126,19 @@ Throughout this document, keys are notated as $component_{owner}^{scheme}$, wher
 | Source     | $`sk_S^{AKEM}`$    | $`pk_S^{AKEM}`$    | SD-APKE | Message  | Incoming+Outgoing | Permanent[^7] | DH-AKEM(X25519, HKDF-SHA256) |                    |
 | Source     | $`sk_S^{PKE}`$     | $`pk_S^{PKE}`$     | SD-PKE  | Metadata | Incoming          | Permanent[^7] | X-Wing (X25519, ML-KEM-768)  |                    |
 
-[^4]: **TODO:** Discussion of whether the newsroom's or the journalist's signing key signs the journalist's fetching key.
```

Comment:
a26f89cb98bb64dee5cebc5844b27221ac52555a documents the latter based on our recent discussions.  Let me know what you think, @rocodes.